src/components/form-item/form-item.jsx

```diff
@@ -0,0 +1,217 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import './form-item.css';
+import ButtonClose from '../button-close';
+
+class FormItem extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showSelectOption: 'invisible',
+    };
+    this.onClickSubmit = this.onClickSubmit.bind(this);
+    this.onClickReset = this.onClickReset.bind(this);
+    this.onClickSelectToggle = this.onClickSelectToggle.bind(this);
+  }
+
+  onClickSubmit() {
```

Comment:
I think the page will be refreshed cause you are using `form`
mb we need to add `event.preventDefault()` in that case
please double check